schemas/proto/foxglove/CompressedVideo.proto

```diff
@@ -0,0 +1,33 @@
+// Generated by https://github.com/foxglove/schemas
+
+syntax = "proto3";
+
+import "foxglove/KeyValuePair.proto";
+import "google/protobuf/timestamp.proto";
+
+package foxglove;
+
+// A single frame of a compressed video bitstream
+message CompressedVideo {
+  // Timestamp of video frame
+  google.protobuf.Timestamp timestamp = 1;
+
+  // Frame of reference for the decoded image. The origin of the frame is the optical center of the camera. +x points to the right in the image, +y points down, and +z points into the plane of the image.
+  string frame_id = 2;
+
+  // Compressed video frame data. For packet-based video codecs this data must begin and end on packet boundaries (no partial packets), and must contain enough video packets to decode exactly one image (either a keyframe or delta frame).
+  bytes data = 3;
+
+  // Indicates whether this video frame is a keyframe or delta frame. In H.26x terminology these are also known as I-frames (intra-coded picture) and P-frames (predicted picture). B-frames are not supported. When this field is set to true, the `metadata` array must be populated as well.
+  bool keyframe = 4;
+
+  // Video metadata used to initialize a video decoder, expressed as an array of key-value pairs. This metadata must be present for every keyframe (when `keyframe` is true), and should be left empty for delta frames. This metadata is modeled after [WebCodecs VideoDecoderConfig](https://www.w3.org/TR/webcodecs/#video-decoder-config), although it can be used to initialize any video decoder. See the See the [WebCodecs Codec Registry](https://www.w3.org/TR/webcodecs-avc-codec-registration/) for more information on the interpretation of each field for a specific codec. The following keys are supported:
+  // 
+  //       * `codec` - A fully qualified codec string describing the video codec. The format for this string is specific to each codec, and is described in the WebCodecs Codec Registry.
+  //       * `description` - Base64-encoded binary data that is specific to each codec. For H.264 this is an AVCDecoderConfigurationRecord (avcC data), and for H.265 this is a HEVCDecoderConfigurationRecord (hvcC data).
```

Comment:
Would this be better called `configuration`? Both of the example you provide contain the word `Configuration` which is more specific than "description" which I first read as a form of human-readable description.